lmdeploy/turbomind/turbomind.py

```diff
@@ -218,6 +219,15 @@ def _from_hf(self, model_source: ModelSource, model_path: str,
                 engine_config.model_format is None:
             engine_config.model_format = 'awq'
 
+        if engine_config.model_format is None:
+            cfg = get_hf_config_content(model_path)
+            quant_config = cfg.get('quantization_config')
+            if quant_config:
+                quant_method = quant_config.get('quant_method')
+                group_size = int(quant_config.get('group_size', 0))
+                if quant_method == 'awq' and group_size == 128:
```

Comment:
I think we should check the version attribute as well.
```
"version": "gemm",
```
There is another version `gemv` we haven't supported yet.

Reply:
Shall we give a warning or assertion error here if it does not meet lmdeploy's requirements?

Reply:
ok. I'll land the code.

Reply:
> it does not meet

Then we'll not set the model_format and at last it'll raise exception. So I think it's not a big deal if we don't address it here.

Reply:
> check the version attribute

@lvhan028 done